docs/docs/index.mdx

```diff
@@ -49,7 +49,8 @@ Time to explore the API
 
 ### Get Help
 
-- [Slack Community](https://join.slack.com/t/magicbell-community/shared_invite/zt-trh6yi84-~jtPqNikvC1m3My_p0WUqw)
+- If you are just starting, join our [GitHub Discussions](https://github.com/orgs/magicbell-io/discussions) to contribute, follow along, and get help from our active community.
+- If you are a power user, access our [Slack Community](https://join.slack.com/t/magicbell-community/shared_invite/zt-trh6yi84-~jtPqNikvC1m3My_p0WUqw) to brag about your integrations on top of powerful features like topics or Slack integrations. Let’s learn from each other!
```

Comment:
Should we link to `magicbell.com/slack` instead? So we have a single place where we need to update the invite token, if it ever changes?